portfolio/src/main/webapp/style.css

```diff
@@ -2,8 +2,67 @@
   margin-left: auto;
   margin-right: auto;
   width: 650px;
+  
 }
 
+
 #greeting-container {
   margin-top: 20px;
 }
+
+h1 {
+    text-align: center;
+    font-size: 30px;
+    color: lightskyblue;
+}
+
+div {
+    font-family: Courier;   
+  text-align: center;
+  padding: 15px;
+  margin: 15px;
+}
+
+#intro-new-zealand {
+    font-size: 20px;
+     border: 5px outset lightskyblue; 
+     background-color: lightyellow;
+
+}
+#margin {
```

Comment:
Indent with 2 spaces each level and space of 1 empty newline between CSS rules
https://google.github.io/styleguide/htmlcssguide.html#Indentation